src/wanderer.ts

```diff
@@ -206,6 +219,40 @@ function guzzlrAbandonQuest() {
   }
 }
 
+function averageYrValue(location: Location) {
+  const monsters = Object.keys(getLocationMonsters(location))
+    .map((m) => toMonster(m))
+    .filter((m) => !m.attributes.includes("LUCKY") && !m.attributes.includes("ULTRARARE"));
+
+  if (monsters.length === 0) {
+    return 0;
+  } else {
+    return sumNumbers(
+      monsters.map((m) =>
+        sumNumbers(
+          itemDropsArray(m)
+            .filter((drop) => drop.type !== "c")
+            .map((drop) => garboValue(drop.drop))
+        )
+      )
+    );
+  }
+}
+
+function bestYrLocation(): { location: Location; value: number } {
+  const validLocations = $locations``
+    .filter(canAdventureOrUnlock)
```

Comment:
So this actually does not feel ideal given the plan here; this is a type of wanderer that is inherently extremely spread out, so we probably don't want to spend meat on, say, a transponder just to YR there. I am open for debate on this point, but actually we should really start to look at the price of these things when we're doing guzzlr as well.

Reply:
so really it's the "or unlock" that gets me

Also, I think we might as well do the combatPercent filtering here, in addition to happening later. We don't want to do math on a really big set of values if we can get away with it.

Reply:
Backups are really the main one that's worth blowing a transponder on, probably. Actually, are transponders expensive? 

8k. At current guzzlr prices that is almost certainly not worth it.

Oh god I just realized what we should do--it's only plats that require these, right? we have it pay attention to unlock price if the platinum isn't capped yet

That is out of scope. But I do think this should probably use a straight canAdv